website/script/fix-path.js

```diff
@@ -30,8 +30,8 @@ try {
   // Fix doc paths
   replace.sync({
     files: './build/ApacheDruid/docs/**/*.html',
-    from: /\/docs\//g,
-    to: '/docs/' + urlVersion + '/',
+    from: /\/docs\/([^">]+\.html)/g,
```

Comment:
How does this work?

Reply:
(By that I mean, how does this fix the bug #8592?)

Reply:
This script 'fixes' the docs made by the Docusaurus build system to be versioned as part of copying the docs into the website.

it used to look for all instances for the string `/docs/` and turn them into `/docs/0.16.0-incubating/` this is pretty crude.

Now as it happens the git repo has links like this `https://github.com/apache/incubator-druid/edit/master/docs/0.16.0-incubating/design/index.md`

So by coincidence the website and the github both have a `docs` directory.

This script would then rewrite the links generated by the `Edit` button causing the bug.

In this fix I tightened up the matcher regexp to only look for links that end in `.html`. This works because it just so happens that all the docs in GH are `.md` and all the docs that are part of the website are `.html`

Reply:
actually... now that I think about it we could make it even stronger by making sure that we have `"/docs/...` instead of `/docs/...`

Reply:
In fact that is all the is needed... we do not need to constrain only to `.html`!